ast.go

```diff
@@ -28,13 +28,23 @@ import (
 type Serializer interface {
 	// Serialize writes the statement/expression to the io.Writer. If an
 	// error is returned the io.Writer may contain partial output.
-	Serialize(w io.Writer) error
+	Serialize(w io.Writer, interpolate bool) error
```

Comment:
If `Serializer` also had `SerializeToString() (string, error)` method, I think `stringSerializer` could be implemented as:

```
type stringSerializer string

func (s stringSerializer) Serialize(w io.Writer) error {
  _, err := w.WriteString(string(s))
 return err
}

func (s stringSerializer) SerializeToString() (string, error) {
  return string(s), nil
}
```